libc/docs/dev/undefined_behavior.rst

```diff
@@ -89,3 +89,24 @@ The C23 standard states that if the value of the ``rnd`` argument of the
 the value of a math rounding direction macro, the direction of rounding is
 unspecified. LLVM's libc chooses to use the ``FP_INT_TONEAREST`` rounding
 direction in this case.
+
+Cached ``getpid/gettid``
+------------------------
+Since version ``2.25``, glibc removes its cache mechanism for ``getpid/gettid`` 
```

Comment:
given the potential downsides, would it be possible to create an option to turn this caching on and off? If you do, then it would be good to mention that flag here.